plugins/woocommerce/templates/myaccount/navigation.php

```diff
@@ -22,7 +22,7 @@
 do_action( 'woocommerce_before_account_navigation' );
 ?>
 
-<nav class="woocommerce-MyAccount-navigation">
+<nav class="woocommerce-MyAccount-navigation" aria-label="Account pages">
```

Comment:
Curious, should this be translated (as with other user-facing text)?

Reply:
@barryhughes made the requested change - makes definately sense.